src/measure_theory/measurable_space.lean

```diff
@@ -258,15 +271,13 @@ begin
   exact (hf ht).inter h.measurable_set.of_compl,
 end
 
-lemma measurable_of_fintype [fintype α] [measurable_singleton_class α] (f : α → β) :
-  measurable f :=
-λ s hs, (finite.of_fintype (f ⁻¹' s)).measurable_set
+omit m
```

Comment:
```suggestion
```
You're ending the section anyways, and properties not needing `m` should go in the previous section.